src/LuaTransformer.ts

```diff
@@ -936,10 +971,14 @@ export class LuaTransformer {
             m => tsHelper.isGetAccessorOverride(m, classDeclarataion, this.checker)
         );
         for (const getter of getOverrides) {
+            const getterName = getter.name !== undefined
+                ? this.expectExpression(this.transformPropertyName(getter.name))
+                : tstl.createAnnonymousIdentifier();
```

Comment:
This isn't right - `getter.name` can't be undefined because the `getter` is actually a `ts.GetAccessorDeclaration` (and if it could, using an anon here wouldn't be the right thing to do). I recommend casting `getOverrides` above:
```ts
const getOverrides = classDeclarataion.members.filter(
    m => tsHelper.isGetAccessorOverride(m, classDeclarataion, this.checker)
) as ts.GetAccessorDeclaration[];
```